models/experimental/functional_bert/README.md

```diff
@@ -0,0 +1,17 @@
+## functional_bert Demo
+## How to Run
+
+Use `pytest --disable-warnings models/experimental/functional_bert/demo/demo.py::test_demo[models.experimental.functional_bert.tt.ttnn_functional_bert-384-8-phiyodr/bert-large-finetuned-squad2-models/experimental/functional_bert/demo/input_data.json]` to run the demo.
+
+If you wish to run the demo with a different input use `pytest --disable-warnings models/experimental/functional_bert/demo/demo.py::test_demo[models.experimental.functional_bert.tt.ttnn_functional_bert-384-8-phiyodr/bert-large-finetuned-squad2-<address_to_your_json_file.json-1>]`. This file is expected to have exactly 8 inputs.
```

Comment:
`<address_to_your_json_file.json-1>` should be without `-1` at the end.

Can you check this. I get the following error using a different input file
```
ERROR: not found: /home/boris/tt-metal/models/experimental/functional_bert/demo/demo.py::test_demo[models.experimental.functional_bert.tt.ttnn_functional_bert-384-8-phiyodr/bert-large-finetuned-squad2-models/experimental/functional_bert/demo/input_data22.json
(no name '/home/boris/tt-metal/models/experimental/functional_bert/demo/demo.py::test_demo[models.experimental.functional_bert.tt.ttnn_functional_bert-384-8-phiyodr/bert-large-finetuned-squad2-models/experimental/functional_bert/demo/input_data22.json' in any of [<Module models/experimental/functional_bert/demo/demo.py>])
```